### ISSUE TITLE
Add semantics on item list changed

### ISSUE THREAD

**Issue by Zigotote** [salim-lachdhaf/searchable_dropdown#673]
**Is your feature request related to a problem? Please describe.**
I'm trying to make my app compliant with accessibility guideline. For that, when the number of displayed items is updated it should be announced by the screen reader.

**Describe the solution you'd like**
Example : 
- I have a dropdown menu with 3 items : pineapple, apple, banana
- I type "a" in the textField : screen reader says "3 items available" (or  something like that)
- I type "ap" in the textField : screen reader says "2 items available" 
- and so on

**Describe alternatives you've considered**
I tried to make this part of accessibility on my side but it is very hard : I can do it if I'm using my own filter, because I known how many items are displayed, but with default filter it's not possible...

**Additional context**

**Comment by salim-lachdhaf** [comment]
is `onItemLoaded` (which returns loaded/showed items) didn't help ?

**Comment by Zigotote** [comment]
I didn't see this props ! I just tried it and it's quite working. I added that : 
```
onItemsLoaded: (items) => SemanticsService.announce(
  "${items.length} available items",
  TextDirection.ltr,
),
```

But it's not read when there is no item. I've got an empty builder and tried to add `SemanticsService.announce` in the builder, but it's not working : 
```
emptyBuilder: (context, searchEntry) {
  SemanticsService.announce(
    "No result", TextDirection.ltr,
  );
  return ListTile(
    title: Text("No result),
  );
},
```

I also tried to wrap my `ListTile` with a `Semantics` widget, with `focused: true` but it doesn't work either.

Would you have an idea on how to achieve this ?

**Comment by salim-lachdhaf** [comment]
i tested using :
```
onItemsLoaded: (items) => SemanticsService.announce(
   print(items.length);
  "${items.length} available items",
  TextDirection.ltr,
),
```
and it works fine using TalkBack, beside idon't see why it would'nt work for empty list and works for non empty list , on both case you get a number.

**Comment by Zigotote** [comment]
Oh, I tried with an other phone and it's working on it... I guess it depends on the OS and some strange things being in conflict.

Well, thank for your help and your quick answers !

**Comment by salim-lachdhaf** [comment]
could you please keep me updated if you find the origin of the issue or a solution that works on all devices (it could be related to `SemanticsService` itself).
